search-sentiments-map/public/scripts/trends-script.js

```diff
@@ -20,11 +20,15 @@ const CLASSNAME_SHOWN = 'shown';
 const CLASSNAME_HIDDEN = 'hidden';
 const NUM_SHOWN = 7;
 
-/** Displays the top trends on the DOM. */
-function setTopTrends() {
+/** 
+ * Displays the top trends on the DOM.
+ * @param {boolean} setNewTrendIsEnabled Boolean for whether or not to change
+ *     the trend that the user is viewing.
```

Comment:
```suggestion
 *  the trend that the user is viewing.
```

Reply:
https://google.github.io/styleguide/jsguide.html#jsdoc

Reply:
I think what Carmen has it correct. See section 7.4

Reply:
I agree with Chenyu it shows that what I originally had was correct in section 7.4.